portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,17 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Praneet's Portfolio</h1>
+      <p>Hello! My name is Praneet and I am a 19 year old from Shrewsbury, MA. I am currently a sophomore at MIT studying computer science.</p>
+      <img src="../images/mw.jpg"></img>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
     </div>
+    <script src="script.js"></script>
```

Comment:
Isn't this usually in <head>?

Reply:
I've been taught to put scripts at the end of body because certain scripts in head wont run until the element itself is present on the screen. I'm not 100% sure about this though.

Reply:
That's good to know!